Cross build of the FMU using docker

I am using OpenModelica v1.19.2 (64-bit), OMSimulator v2.1.1.post159-g606161d-mingw under Windows 64bit. I created a fmu and want win64 and linux64 binaries. I ticked on x86_64-linux-gnu docker build and already have docker installed on it.

When I export the FMU, I am getting the following error:

```
Error building simulator. Build log: docker run  --rm -w /fmu -v ...f:/fmu multiarch/crossbuild sh -c "cd "/fmu/model.fmutmp/sources" && ./configure --host='x86_64-linux-gnu' CFLAGS='-Os' CPPFLAGS=-I/fmu/fmiInclude LDFLAGS= && make -j12 nozip":
...
sh: 1: ./configure: not found
```

I think the issue is related to #7458.

Thanks  for helping @AnHeuermann. 
Adding the compilation flag for CMake build helped to export the FMU. However, now there are 2 issues. 

1. FMU Export works fine with Explicit Euler solver but with CVode I got the following error.

```
[ 35%] Building C object CMakeFiles/Modelica_Electrical_Analog_Examples_Rectifier.dir/simulation/simulation_omc_assert.c.o
/fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/simulation_omc_assert.c: In function 'omc_assert_simulation':
/fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/simulation_omc_assert.c:115:1: warning: 'noreturn' function does return
 } 
 ^
/fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/simulation_omc_assert.c: In function 'omc_assert_simulation_withEquationIndexes':
/fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/simulation_omc_assert.c:123:1: warning: 'noreturn' function does return
 } 
 ^
[ 36%] Building C object CMakeFiles/Modelica_Electrical_Analog_Examples_Rectifier.dir/simulation/solver/cvode_solver.c.o
In file included from /fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/solver/cvode_solver.c:36:0:
/fmu/Modelica_Electrical_Analog_Examples_Rectifier.fmutmp/sources/simulation/solver/cvode_solver.h:35:26: fatal error: omc_config.h: No such file or directory
   #include "omc_config.h" 
                          ^
compilation terminated.
```

2. If I select, Native and x86_64-linux, it is giving me following error. I am using Windows 11 64-bit operating system, x64-based processor. 
```
Error building simulator. Build log: Unknown/unsupported platform "static " for CMake FMU build
```

I am using the OpenModellica 1.19.2 (Stable Release). I want Co-Simulation FMU with version 2.0 in linux64 & win64 binaries. and cvode solver with blackbox filter.

--- a/OMCompiler/SimulationRuntime/fmi/export/buildproject/CMakeLists.txt.in
+++ b/OMCompiler/SimulationRuntime/fmi/export/buildproject/CMakeLists.txt.in
@@ -52,9 +52,10 @@ add_library(${FMU_NAME} SHARED
 target_link_libraries(${FMU_NAME} PRIVATE Threads::Threads)
 
 target_include_directories(${FMU_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_include_directories(${FMU_NAME} PRIVATE ${FMI_INTERFACE_HEADER_FILES_DIRECTORY})
+target_include_directories(${FMU_NAME} PRIVATE ${FMI_INTERFACE_HEADER_FILES_DIRECTORY}
+                                               sundials)
 
-target_compile_definitions(${FMU_NAME} PRIVATE OMC_MINIMAL_RUNTIME=1;DOMC_FMI_RUNTIME=1;CMINPACK_NO_DLL)
+target_compile_definitions(${FMU_NAME} PRIVATE OMC_MINIMAL_RUNTIME=1;OMC_FMI_RUNTIME=1;CMINPACK_NO_DLL)
 
 install(TARGETS ${FMU_NAME}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}

